### DISCUSSION TITLE
Parallel Tests with Atata + Specflow

### DISCUSSION THREAD

**Discussion by gustavomoises** [atata-framework/atata#592] [Q&A]
I am trying to parallelize tests using Atata + Specflow. I am using as example the AtataSamples.ParallelTestsReusingDrivers. The driver pool is working but the tests are not running in parallel.

Does anyone have a simple parallel test example with Atata+Specflow that I can use as reference?

Thanks,

Gustavo

**Comment by YevgeniyShunevych** [comment]
What testing library do you use with SpecFlow: NUnit, xUnit, MSTest? It's their responsibility to run tests in parallel.
